lib/Doctrine/Migrations/Configuration/Connection/ConnectionLoaderInterface.php

```diff
@@ -17,6 +18,8 @@ interface ConnectionLoaderInterface
     /**
      * Read the input and return a Connection, returns null if the config
      * is not supported.
+     *
+     * @throws ConnectionNotSpecified
      */
-    public function chosen() : ?Connection;
+    public function getConnection() : Connection;
```

Comment:
Do we need the `get` prefix here or can we drop it?

Reply:
For consistency with the res of doctrine libraries will keep it as `getConnection` ... see ORM, DBAL, and this library too..